{"name":"Integrity constraint violation","message":"SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \"category_pkey\"\nDETAIL:  Key (id)=(1) already exists.\nThe SQL being executed was: INSERT INTO \"category\" (\"slug\", \"title\", \"enabled\") VALUES ('telefony', 'Телефоны', TRUE) RETURNING \"id\"","code":"23505","type":"yii\\db\\IntegrityException","file":"/app/vendor/yiisoft/yii2/db/Schema.php","line":678,"stack-trace":["#0 /app/vendor/yiisoft/yii2/db/Command.php(1307): yii\\db\\Schema->convertException(Object(PDOException), 'INSERT INTO \"ca...')","#1 /app/vendor/yiisoft/yii2/db/Command.php(1168): yii\\db\\Command->internalExecute('INSERT INTO \"ca...')","#2 /app/vendor/yiisoft/yii2/db/Command.php(424): yii\\db\\Command->queryInternal('fetch', NULL)","#3 /app/vendor/yiisoft/yii2/db/pgsql/Schema.php(646): yii\\db\\Command->queryOne()","#4 /app/vendor/yiisoft/yii2/db/ActiveRecord.php(600): yii\\db\\pgsql\\Schema->insert('{{%category}}', Array)","#5 /app/vendor/yiisoft/yii2/db/ActiveRecord.php(566): yii\\db\\ActiveRecord->insertInternal(NULL)","#6 /app/vendor/yiisoft/yii2/db/BaseActiveRecord.php(678): yii\\db\\ActiveRecord->insert(true, NULL)","#7 /app/vendor/yiisoft/yii2/rest/CreateAction.php(52): yii\\db\\BaseActiveRecord->save()","#8 [internal function]: yii\\rest\\CreateAction->run()","#9 /app/vendor/yiisoft/yii2/base/Action.php(94): call_user_func_array(Array, Array)","#10 /app/vendor/yiisoft/yii2/base/Controller.php(181): yii\\base\\Action->runWithParams(Array)","#11 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\\base\\Controller->runAction('create', Array)","#12 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\\base\\Module->runAction('category/create', Array)","#13 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\\web\\Application->handleRequest(Object(yii\\web\\Request))","#14 /app/api/web/index-test.php(28): yii\\base\\Application->run()","#15 {main}"],"error-info":["23505",7,"ERROR:  duplicate key value violates unique constraint \"category_pkey\"\nDETAIL:  Key (id)=(1) already exists."],"previous":{"name":"Exception","message":"SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \"category_pkey\"\nDETAIL:  Key (id)=(1) already exists.","code":"23505","type":"PDOException","file":"/app/vendor/yiisoft/yii2/db/Command.php","line":1302,"stack-trace":["#0 /app/vendor/yiisoft/yii2/db/Command.php(1302): PDOStatement->execute()","#1 /app/vendor/yiisoft/yii2/db/Command.php(1168): yii\\db\\Command->internalExecute('INSERT INTO \"ca...')","#2 /app/vendor/yiisoft/yii2/db/Command.php(424): yii\\db\\Command->queryInternal('fetch', NULL)","#3 /app/vendor/yiisoft/yii2/db/pgsql/Schema.php(646): yii\\db\\Command->queryOne()","#4 /app/vendor/yiisoft/yii2/db/ActiveRecord.php(600): yii\\db\\pgsql\\Schema->insert('{{%category}}', Array)","#5 /app/vendor/yiisoft/yii2/db/ActiveRecord.php(566): yii\\db\\ActiveRecord->insertInternal(NULL)","#6 /app/vendor/yiisoft/yii2/db/BaseActiveRecord.php(678): yii\\db\\ActiveRecord->insert(true, NULL)","#7 /app/vendor/yiisoft/yii2/rest/CreateAction.php(52): yii\\db\\BaseActiveRecord->save()","#8 [internal function]: yii\\rest\\CreateAction->run()","#9 /app/vendor/yiisoft/yii2/base/Action.php(94): call_user_func_array(Array, Array)","#10 /app/vendor/yiisoft/yii2/base/Controller.php(181): yii\\base\\Action->runWithParams(Array)","#11 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\\base\\Controller->runAction('create', Array)","#12 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\\base\\Module->runAction('category/create', Array)","#13 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\\web\\Application->handleRequest(Object(yii\\web\\Request))","#14 /app/api/web/index-test.php(28): yii\\base\\Application->run()","#15 {main}"]}}